coffee_boat/captain.py

```diff
@@ -171,30 +178,24 @@ def _launch_conda_ship(self):
         new_args = "--files {0},{1} {2}".format(zip_target, runner_script_path, old_args)
         print("using {0} as python arguments".format(new_args))
         os.environ["PYSPARK_SUBMIT_ARGS"] = new_args
+        # Handle active/already running contexts.
+        sc = pyspark.context.SparkContext._active_spark_context
+        if sc is not None:
+            print("Adding {0} & {1} to existing sc".format(zip_target, runner_script_path))
```

Comment:
A question for @holdenk @rgbkrk: which versions of Python will coffee_boat run? If Python 3.6 or higher, we can go with the far simpler f-strings. If this needs to run on 3.4 or 2.7, whelp, we are stuck with `format(blah, blah, blah)

Reply:
Probably for the best adoption, I'd opt for 2.7+ and 3.4+.